src/code42cli/cmds/alerts.py

```diff
@@ -272,12 +272,15 @@ def send_to(
     use_checkpoint,
     or_query,
     certs,
+    ignore_cert_validation,
     **kwargs
 ):
     """Send alerts to the given server address.
 
     HOSTNAME format: address:port where port is optional and defaults to 514.
     """
+    if ignore_cert_validation:
```

Comment:
I tried putting this in a callback, having it set the other param automatically, but couldn't get it to work right

Reply:
I can create custom command type to handle it though

Reply:
Or include it in the CLIState